test/Microsoft.AspNetCore.Razor.Design.Test/IntegrationTests/CleanProjectIntegrationTest.cs

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Threading.Tasks;
+using Xunit;
+
+namespace Microsoft.AspNetCore.Razor.Design.IntegrationTests
+{
+    public class CleanProjectIntegrationTest : MSBuildIntegrationTestBase
+    {
+        [Fact]
+        [InitializeTestProject("SimpleMvc")]
+        public async Task CleanProject_RunBuild()
+        {
+            var result = await DotnetMSBuild("Restore;Build"); // Equivalent to dotnet build
```

Comment:
By the way, one suggestion for improvement. Restore is now a first-class MSBuild feature: dotnet build = dotnet msbuild /restore /:Build